soc/realtek/ec/rts5912/debug_swj.c

```diff
@@ -12,6 +12,8 @@
 
 PINCTRL_DT_DEFINE(SWJ_NODE);
 
+#if DT_NODE_HAS_STATUS_OKAY(SWJ_NODE)
```

Comment:
You should exclude this file using Kconfig.

In Kconfig in this directory.
```
config RTS5912_DEBUG_SWJ
    bool "Enable serial wire JTAG"
    default y
    depends on DT_HAS_SWJ_CONNECTOR_ENABLED
    help
      Enables the serial wire JTAG connection on the RTS5912 EC.
```

In CMakeFiles.txt
```
zephyr_sources_ifdef(CONFIG_RTS5912_DEBUG_SWJ debug_swj.c)
```